umbraco/Umbraco-CMS
# returns node name instead id if value is a udi

**Please note:** I'm not sure if the right way of getting the node name is using `node.variants[0].name` but it's the only way I found. This is in line 653.

### Prerequisites


this fixes #5545 

### Description
If the value of the custom property is a Udi, it'll get the node and return its name instead the raw value.

### How to test
Create a container doctype. Allow in root. Enable list view.
Create a DocType with a content picker. Allow this as child for the previous container.
Add the picker property to the list view of the container.
Create a couple of pages using that doctype. Configure the picker in one of them selecting the other one.
Navigate to the container's list view. You should get the name of the selected node instead its Id.
